### ISSUE TITLE
Desktop App Not Finding Xome Device

### ISSUE THREAD

**Issue by mrobinsonasl1** [hoobs-org/HOOBS#1463]
![tempImagel3hwpv](https://user-images.githubusercontent.com/19247432/129500623-e0106692-d756-41c5-9609-e74a5a021ee2.gif)

I tried the advanced field and put in the ip address still nothing.

**Comment by mkellsy** [comment]
What device are you using?
The Raspberry Pi should not have a red LED.
Is the red LED showing?

**Comment by mrobinsonasl1** [comment]
I am using a Home Homebridge and it is  displaying a green light,

I done the first steps, I get to the point of adding my WiFI network I add out with the password to access it. and I went on the web and typed Hoobs.Local it says it won't work I tried the help suggestions for Hoobs I think it worked once I open Hoobs app on my mac. 

This window above pictured  is what I get. I tried advance and put in the ip address of the device and port, I seen a command I could use to show me the IP address in terminal. I used the IP address it gave me and the interface on Hoobs said its invalid IP address.
